src/Infrastructure/Persistence/Migrations/20241022032926_AddHeroesAndTeams.cs

```diff
@@ -16,7 +16,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 columns: table => new
                 {
                     Id = table.Column<Guid>(type: "uniqueidentifier", nullable: false),
-                    Name = table.Column<string>(type: "nvarchar(100)", maxLength: 100, nullable: false),
+                    Name = table.Column<string>(type: "nvarchar(max)", nullable: false),
```

Comment:
What's caused the length of these columns to change?

Reply:
As per our conversation, I suspect this is due to the change in EF core tools version. As discussed I removed the migration and ran it again using 9 rc and confirmed there is no difference with this version.